cocos2d/core/sprites/CCSpriteBatchNode.js

```diff
@@ -58,89 +59,63 @@
 cc.SpriteBatchNode = _ccsg.Node.extend(/** @lends cc.SpriteBatchNode# */{
```

Comment:
SpriteBatchNode is now a pure node which detects the texture of children added. No other funtionalities any more